config/config.php

```diff
@@ -1,7 +1,20 @@
 <?php
 
+use Julienbourdeau\RouteUsage\Http\Middleware\Authorize;
+
 return [
 
+    /*
+    |--------------------------------------------------------------------------
+    | Route Usage Path
+    |--------------------------------------------------------------------------
+    |
+    | This is the URI path where this package will be accessible from.
+    | Feel free to change this path to anything you like.
+    |
+    */
+    'path' => env('ROUTE_USAGE_PATH', 'route-usage'),
```

Comment:
I'd rather not add this for now. I prefer _convention over configuration_ and I don't really see the point.

Reply:
@julienbourdeau the point is what if your package will add more routes? It would be nice if all of the package route will be on the based path like `/route-usage` and any child route will be below it like `route-usage/{id}`

Reply:
Here is an example

![image](https://user-images.githubusercontent.com/10015302/67726586-44675c00-fa21-11e9-9af1-4b6860715e06.png)

Sorry it didn't return 403 in my sample its because we are running on SPA in my based path `/` but for a regular Laravel app it should show you the 403 forbidden page

Reply:
So because of this the user of the package can easily change the default route name to anything they want :)

Reply:
It solves one issue for sure,: if you already have a route matching `/route-usage`.

I can't make up my mind on this yet. I know by experience that it's always easy to add feature at the beginning but it usually makes the code harder to maintain over time.

> what if your package will add more routes?
There are no plan to add more routes for now. And still I'd rather add this feature when it's necessary.